### PR TITLE
Revert "Revert "reduce image size for animated .gif files and convert to .webp""

### PR DESCRIPTION
**Pull request by susanev**
Reverts pulumi/pulumi-hugo#2212

### CODE REVIEW

File: themes/default/content/blog/announcing-crossguard-preview/index.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ const bucket = new aws.s3.Bucket("my-bucket", {
 });
 ```
 
-![Demo of Policy as Code](pac-demo.gif)
+![Demo of Policy as Code](pac-demo.webp)
```
Comment (komalali) [inline themes/default/content/blog/announcing-crossguard-preview/index.md:85]: Looks like this file is actually much larger as `webp`
Screen Shot 2022-11-10 at 4 40 36 PM

**Comment by cnunciato** [comment]
Can this be closed?

**Comment by susanev** [comment]
@cnunciato sure if you dont want to try again you can close it

**Comment by cnunciato** [comment]
Ok thanks, I'll close for now, then. Sounds like all is well for the time being.

**Comment by susanev** [comment]
@cnunciato im not sure if all is well, i think we are still pretty close to being at the limit.

**Comment by cnunciato** [comment]
@susanev I interpreted [your comment here](https://github.com/pulumi/docs/issues/8220#issuecomment-1310605145) as suggesting the issue had been sufficiently addressed for now. Perhaps I misread; will take a closer look. Meantime, Zack is considering options for dealing with this limitation more permanently.

**Comment by cnunciato** [comment]
@susanev Ok yeah, looks like we don't actually have much room here after all. I'll see what I can do.

**Comment by susanev** [comment]
@cnunciato sry for not being clear, that change mentioned in that comment was reverted. ill reopen the issue.

**Comment by cnunciato** [comment]
Ok, this should help win back some space: https://github.com/pulumi/pulumi-hugo/pull/2235

**Pull request closed without merging**